File: hw/alban/u3/hw22MazeDiagramClass/Maze.js/view/ASCIIPrinter.js
import { MazeRenderer } from './MazeRenderer.js';
export class ASCIIPrinter extends MazeRenderer {
    render(maze) {
        console.log("[View ASCII] Printing maze representation using text characters:");
        console.log("+---+---+---+");
        console.log("S   |       |");
        console.log("+---+   +---+");
        console.log("|       |   E");
        console.log("+---+---+---+");
    }
}

File: hw/alban/u3/hw22MazeDiagramClass/Maze.java/MazeJava/Maze/src/main/java/ec/edu/espe/Maze/view/ASCIIPrinter.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package ec.edu.espe.Maze.view;
import ec.edu.espe.Maze.model.Maze;

/**
 *
 * @author Angie Nacato, Error 404, @ESPE
 */
public class ASCIIPrinter implements MazeRenderer {
    @Override
    public void render(Maze maze) {
        System.out.println("Renderizando laberinto en consola [" + maze.getWidth() + "x" + maze.getHeight() + "]");
    }
}